dataprofiler/reports/graphs.py

```diff
@@ -7,25 +7,42 @@
 import seaborn as sns
 
 
-def plot_histograms(profiler, columns=None):
+def plot_histograms(profiler, columns=None, column_inds=None):
     """
     Take a input of StructuredProfiler class and a list of specified column
     names and then plots the histograms of those that are int or float
     columns.
 
     :param profiler: StructuredProfiler variable
     :type profiler: StructuredProfiler
-    :param columns: list of column names to be plotted
-    :type columns: list
-    :return: returns fig
-    :rtype: fig
+    :param columns: List of column names to be plotted. Can only specify columns
+        or column_inds, but not both
+    :type columns: list[Union[int,str]]
+    :param column_inds: List of column indexes to be plotted
+    :type column_inds: list[int]
```

Comment:
Why is this necessary, given you can control the `columns` that are being passed in?

Reply:
columns may have duplicates, but column indexes are unique, updated naming to fix